### PR TITLE
Minify and only import the parts of lodash we care about

### PR DESCRIPTION
**Pull request by PepperTeasdale**
TODO:

- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests pass
- [x] Update CHANGELOG.md with your change
- [x] If this was a change that affects the public API, update the README

### CODE REVIEW

File: src/buildValidations.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
-import { isArray, pickBy, get, startCase, forEach } from 'lodash';
+import isArray from 'lodash/isArray';
```
Comment (jdanz) [inline src/buildValidations.js]: If it were to improve build size significantly, is there a way we can get rid of any of these imports as well?

File: src/buildValidations.spec.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { some } from 'lodash';
+import some from 'lodash/some';
```
Comment (jdanz) [inline src/buildValidations.spec.js]: Can we just get rid of this entirely? Looks like we only use it for one test

Comment (PepperTeasdale) [reply src/buildValidations.spec.js]: Tests won't be in the final built javascript, so this should not affect anything including this package

Comment (jdanz) [reply src/buildValidations.spec.js]: 👍

File: src/buildValidations.js (hunk 1 above)
Comment (jdanz) [reply src/buildValidations.js]: 👍

Comment (danielnovograd) [reply src/buildValidations.js]: `Array.isArray` is good

**Pull request merged**